optimum/habana/transformers/training_args.py

```diff
@@ -170,6 +172,11 @@ class GaudiTrainingArguments(TrainingArguments):
         metadata={"help": ("Set value of 'dynamic' parameter for torch.compile.")},
     )
 
+    inline_inbuilt_nn_modules: Optional[bool] = field(
+        default=None,
```

Comment:
If I understand correctly, this should be True for all models except ALBERT right?
So we might as well set the default to True no?

Reply:
We want to keep the default value of this switch consistent with different versions of Pt. For example, pt 2.4 defaults to False, while pt 2.5 defaults to True. Therefore, using None here means not applying any configuration to maintain alignment with the respective version's defaults.